paths/job_locales/create.yaml

```diff
@@ -73,4 +73,18 @@ requestBody:
               type: string
             example:
             - abcd1234cdef1234abcd1234cdef1234
+          team_ids:
+            description: Array of team ids to be assigned to the job locale as translators
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          reviewer_team_ids:
+            description: Array of team ids to be assigned to the job locale as reviewers
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
 x-cli-version: '2.5'
```

Comment:
I think this needs to be set to an CLI release that will include the new params. So maybe 2.6.7 if that's the next release